firefox-ios/Client/Frontend/Settings/ThemeSettings/ThemeMiddleware.swift

```diff
@@ -79,10 +78,10 @@ class ThemeManagerMiddleware: ThemeManagerProvider {
         themeManager.setAutomaticBrightness(isOn: enabled)
     }
 
-    func updateManualTheme(_ theme: ThemeType) {
-        let isLightTheme = theme == .light
+    func updateManualTheme(_ newTheme: ThemeType) {
+        let isLightTheme = newTheme == .light
         legacyThemeManager.current = isLightTheme ? LegacyNormalTheme() : LegacyDarkTheme()
-        themeManager.changeCurrentTheme(isLightTheme ? .light : .dark)
+        themeManager.changeCurrentTheme(newTheme)
```

Comment:
logic changed here in which we don't default to dark theme, are we okay with this update?

Reply:
Well, this should default to whatever's called, really, as opposed to having a random default. It's cleaner this way and the logic is easier to follow, imo.